.pre-commit-config.yaml

```
@@ -6,52 +6,64 @@ repos:
       rev: v6.0.0
       hooks:
           - id: check-added-large-files
-            args: ["--maxkb=1024"]
+            args: [ "--maxkb=1024" ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)